doc/team_practices.md

```diff
@@ -82,24 +170,46 @@ Example flow:
     git clone git@github.com:18F/C2.git
     cd C2
     git checkout -b qa-123456-fix-timezones
```

Comment:
ah yup - changed to remove concept of numbers since that is a tracker convention -- will change in all places thanks for catching!